cylc/flow/xtrigger_mgr.py

```diff
@@ -122,7 +122,6 @@ def __init__(
         proc_pool: SubProcPool = None,
         suite_run_dir: str = None,
         suite_share_dir: str = None,
-        suite_work_dir: str = None,
```

Comment:
Not used. These arguments must be created with the key (`, *, ...`), which helped searching for its usage. Appeared to be safe to remove :+1: